source/documentation/16-support-contact-and-more-information.md

```diff
@@ -31,77 +14,31 @@ Incident severity is defined below:
 | P3 | Significant | Users experiencing intermittent or degraded service due to platform issue | 1 business day or next working day | 2 business days |
 | P4 | Minor | Component failure that is not immediately service impacting | 2 business days | weekly |
```

Comment:
@katiebates @heathd  can we add something here along the lines of "normal BAU functions" e.g. switching an account to live